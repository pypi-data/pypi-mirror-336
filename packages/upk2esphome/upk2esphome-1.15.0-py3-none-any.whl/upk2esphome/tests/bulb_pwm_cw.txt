{
  "manufacturer": "Kobi",
  "name": "Oprawa Smart LED Moon",
  "key": "keytg5kq8gvkv9dh",
  "ap_ssid": "SmartLife",
  "github_issues": [
    264
  ],
  "image_urls": [
    "https://tuya-cloudcutter.github.io/images/kobi-oprawa-smart-led-moon.jpg"
  ],
  "profiles": [
    {
      "slug": "oem-bk7231s-light-ty-2.9.16-sdk-1.0.8-40.00",
      "name": "2.9.16 - BK7231T",
      "type": "CLASSIC",
      "sub_name": "oem_bk7231s_light_ty",
      "icon": "lightbulb-outline"
    }
  ],
  "schemas": {
    "0000021ioj": [
      {
        "type": "obj",
        "mode": "rw",
        "property": {
          "type": "bool"
        },
        "id": 20
      },
      {
        "type": "obj",
        "mode": "rw",
        "property": {
          "range": [
            "white",
            "colour",
            "scene",
            "music"
          ],
          "type": "enum"
        },
        "id": 21
      },
      {
        "type": "obj",
        "mode": "rw",
        "property": {
          "min": 10,
          "max": 1000,
          "scale": 0,
          "step": 1,
          "type": "value"
        },
        "id": 22
      },
      {
        "type": "obj",
        "mode": "rw",
        "property": {
          "min": 0,
          "max": 1000,
          "scale": 0,
          "step": 1,
          "type": "value"
        },
        "id": 23
      },
      {
        "type": "obj",
        "mode": "rw",
        "property": {
          "type": "string",
          "maxlen": 255
        },
        "id": 25
      },
      {
        "type": "obj",
        "mode": "rw",
        "property": {
          "min": 0,
          "max": 86400,
          "scale": 0,
          "step": 1,
          "type": "value"
        },
        "id": 26
      },
      {
        "type": "obj",
        "mode": "wr",
        "property": {
          "type": "string",
          "maxlen": 255
        },
        "id": 28
      }
    ]
  },
  "device_configuration": {
    "Jsonver": "1.1.8",
    "brightmax": 100,
    "brightmin": 10,
    "c_lv": 1,
    "c_pin": 7,
    "cagt": 20,
    "category": "0502",
    "cmod": "cw",
    "colormax": 100,
    "colormin": 10,
    "crc": 100,
    "ctrl_lv": 1,
    "ctrl_pin": 14,
    "cwmaxp": 100,
    "cwtype": 0,
    "defbright": 50,
    "defcolor": "c",
    "deftemp": 0,
    "dmod": 0,
    "module": "WB3L",
    "onoffmode": 0,
    "pmemory": 1,
    "prodagain": 0,
    "pwmhz": 1000,
    "remdmode": 0,
    "rgbt": 0,
    "rstbr": 50,
    "rstcor": "c",
    "rstnum": 3,
    "rsttemp": 0,
    "title20": 1,
    "w_lv": 1,
    "w_pin": 6,
    "wfcfg": "spcl",
    "wfct": 3,
    "wt": 20
  },
  "slug": "kobi-oprawa-smart-led-moon",
  "image_url": "https://tuya-cloudcutter.github.io/images/thumbs/kobi-oprawa-smart-led-moon.jpg"
}
