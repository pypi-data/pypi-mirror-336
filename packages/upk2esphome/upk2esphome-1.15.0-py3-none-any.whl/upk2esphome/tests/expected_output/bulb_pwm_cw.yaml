output:
  - platform: libretiny_pwm
    id: output_cold
    pin: P7
  - platform: libretiny_pwm
    id: output_warm
    pin: P6

light:
  - platform: cwww
    id: light_cwww
    name: Light
    cold_white_color_temperature: 6500 K
    warm_white_color_temperature: 2700 K
    cold_white: output_cold
    warm_white: output_warm
